packages/shared/src/components/cards/MarketingCta/common.tsx

```diff
@@ -22,8 +22,8 @@ export enum MarketingCtaVariant {
 }
 
 export interface MarketingCta {
-  campaignId: string;
-  createdAt: Date;
+  campaignId?: string;
+  createdAt?: Date;
```

Comment:
Made this optional to understand whether the promotion came from the marketing team so we can apply the necessary tracking only if applicable.

Reply:
Thoughts? cc @rebelchris @omBratteng @nensidosari

Reply:
Yeah I think that's good

Reply:
Checked with product/ido. They want to have a campaign id for each.